packages/block-editor/src/components/iframe/use-compatibility-styles.js

```diff
@@ -70,10 +70,6 @@ export function useCompatibilityStyles() {
 				}
 
 				if ( matchFromRules( cssRules ) ) {
-					// Display warning once we have a way to add style dependencies to the editor.
-					// See: https://github.com/WordPress/gutenberg/pull/37466.
```

Comment:
We no longer need this comment because we added the warning in https://github.com/WordPress/gutenberg/pull/50091.